test/common/initModule.js

```diff
@@ -20,7 +20,6 @@ var ed = require('../../helpers/ed');
 var jobsQueue = require('../../helpers/jobsQueue');
 var Transaction = require('../../logic/transaction.js');
 var Account = require('../../logic/account.js');
-var Sequence = require('../../helpers/sequence.js');
 
 var modulesLoader = new function () {
 
```

Comment:
I see arguments `scope` and `cb` disordered among the different functions. Shall we always follow the same patter? `(scope, cb)`

Reply:
we should, but it is handy in case of a non-required param. Also while using es6 default params features it is not possible to specify them before required ones

Reply:
I get your point. Then we wait till the milestone's end